### PR TITLE
XX 179 | SFMultiSelect Component

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Components/SFMultiSelect/SFMultiSelect.stories.tsx [hunk 1]
```diff
@@ -67,38 +67,20 @@ export default {
...
     }
   }
-} as Meta;
+} as ComponentMeta<typeof SFMultiSelect>;
 
-const Template: Story<SFMultiSelectProps> = (args) => (
-  <SFMultiSelect {...args} />
-);
+const Template: ComponentStory<typeof SFMultiSelect> = (args) => {
+  const [value, setValue] = React.useState<string[]>(args.value ?? []);
```
Comment (martinweingart) [inline src/Components/SFMultiSelect/SFMultiSelect.stories.tsx:75]: Don't use state, just storybook args

**Review by martinweingart — APPROVED** [review]
LGTM

File: src/Components/SFMultiSelect/SFMultiSelect.stories.tsx (hunk 1 above)
Comment (martinweingart) [inline src/Components/SFMultiSelect/SFMultiSelect.stories.tsx:75]: Don't use state, just storybook args

**Review by martinweingart — CHANGES_REQUESTED** [review]

File: src/Components/SFMultiSelect/SFMultiSelect.stories.tsx (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Components/SFMultiSelect/SFMultiSelect.stories.tsx]: Done, removed

**Review by martinweingart — APPROVED** [review]
LGTM

**Pull request merged**